nengo/tests/test_processes.py

```diff
@@ -170,8 +170,9 @@ def test_whitesignal_y0(Simulator, seed, y0, d):
     with Simulator(model, seed=seed) as sim:
         sim.run(t)
     values = sim.data[up]
+    error = np.min(abs(y0 - values), axis=0)
 
-    assert np.allclose(values[0, :], y0, atol=1e-3)
+    assert ((y0 - error <= values[0, :]) & (values[0, :] <= y0 + error)).all()
```

Comment:
That part of the message was referring to the `error` calculation. For short signals with high frequencies there is no strict guarantee that we'll get anything close to `y0`. This checks that we at least got as close as we possibly could, by making it sure it matches the available minimum (note each dimension will match either the lower or upper bound).